code-challenges.test.js

```diff
@@ -15,15 +15,56 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
-// Example input: 6
+var num1 = 6
 // Expected output: [1, 1, 2, 3, 5, 8]
 
-// Example input: 10
+var num2 = 10
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
+// Create Jest test using syntax example from syllubus with describe, it, and expect statements.
+describe("fibFunction", () => {
+  it("takes a number and returns array in Fibonacci sequence", () => {
+// set expect to equal an array of the length of the number, with the values being the Fibonacci sequence
+    expect(fibFunction(num1)).toEqual([1, 1, 2, 3, 5, 8])
+    expect(fibFunction(num2)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55])
+  })
+})
+
+// Run the test to make sure you receive red error message that function name is not defined.
+// Note: Jest test produced desired "red message"
 
 // b) Create the function that makes the test pass.
 
+const fibFunction = (num) => {
+
+// create array to return. Set first and second index to 1 and 1
+var fib = [1, 1];
+// create if statement for edge cases where number is less than 3
+if (num < 3){
+  return [];
+// create else statement for other cases
+} else {
+// create for loop to iterate once per value of number
+for (var i = 2; i < num; i++) {
+
+  // Fibonacci sequence formula: Xn = Xn-1 + Xn-2
+  let first = fib[i - 2];
+  let second = fib[i - 1];
+  fib[i] = first + second;
+// push the next value into the array
```

Comment:
I'm wondering if the issue is here on line 53. I'm curious is redefining `fib[i]` is causing issues.